src/scss/includes/panels/poi_panel.scss

```diff
@@ -299,7 +299,11 @@ $CARD_PADDING: 8px;
   padding-left: 2px;
   height: 25px;
   font-size: 12px;
-  color: $secondary_text;
```

Comment:
I think we need to keep this color (by default without hover)

Reply:
yep, removed it and haven't seen the difference, now i see it.